### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Capoverde**
[DEMO LINK](https://Capoverde.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://Capoverde.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="page__header">
+      <a href="./index.html" class="logo"><img src="./images/logo.png" alt="logo"></a>
```
Comment (choeqq) [inline src/index.html:13]: fomartting seems to be little off, are you using autoformatting? 🤔

Comment (choeqq) [inline src/index.html:13]: please wrap `<a>` tag with `<div>` - it is way easier to move around 😉

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,73 @@
-body {
+@import url(https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic);
```
Comment (choeqq) [inline src/style.css:1]: you can also import fonts in `<head>`, but that also works

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,73 @@
...
+
+*,
+*::before,
+*::after {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
```
Comment (choeqq) [inline src/style.css:11]: nice usage of CSS variables 👏

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,73 @@
...
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+  line-height: var(--header-height);
+  text-decoration: none;
+  color: inherit;
+  height: 100%;
+  display: inline-block;
+}
+
+.nav__item:hover > .nav__link {
```
Comment (choeqq) [inline src/style.css:56]: I don't think we need to be that specific with those selectors, just put the hover on concrete element 😄

**Review by choeqq — CHANGES_REQUESTED** [review]
Just little changes to apply 😄

File: src/index.html (hunk 1 above)
Comment (Capoverde) [reply src/index.html:13]: I've been trying to disable it. :)

Comment (Capoverde) [reply src/index.html:13]: You meen div > a > img?

Comment (choeqq) [reply src/index.html:13]: why exactly? code is more readable when correctly formatted 😄

File: src/style.css (hunk 4 above)
Comment (Capoverde) [reply src/style.css:56]: I thought that the more specific the better...Ofcourse I will fix that :)

File: src/index.html (hunk 1 above)
Comment (choeqq) [reply src/index.html:13]:
```suggestion
      <div class="some_class"><a href="./index.html" class="logo"><img src="./images/logo.png" alt="logo"></a></div>
```

Comment (Capoverde) [reply src/index.html:13]: And the core word is 'rorrectly' :)

File: src/index.html [hunk 5]
```diff
@@ -6,11 +6,17 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```
Comment (choeqq) [inline src/index.html]: you pasted it twice, you can remove one of those

**Review by choeqq — APPROVED** [review]
Nicely done, all test are passing - Approved! ✅

File: src/index.html (hunk 5 above)
Comment (Capoverde) [reply src/index.html]: I saw it and I did that. I don't know where is my latest commit...
Tell me what should I do? Should I close PR and send another?

Comment (choeqq) [reply src/index.html]: just update the PR on develop branch and push it here, you can also remove `header-height` from variables - we're using it only once, so there is no point in creating variable for that 😉

**Review by mvjl000 — APPROVED** [review]

**Review by d-wasilewski — CHANGES_REQUESTED** [review]
test

**Review by d-wasilewski — CHANGES_REQUESTED** [review]
2

**Review by d-wasilewski — APPROVED** [review]

**Pull request closed without merging**